linode_api4/objects/networking.py

```diff
@@ -56,8 +92,39 @@ def to(self, linode):
 
         if not isinstance(linode, Instance):
             raise ValueError("IP Address can only be assigned to a Linode!")
+
         return {"address": self.address, "linode_id": linode.id}
 
+    def ip_addresses_share(self, ips, linode_id):
+        """
+        Configure shared IPs.
+        """
+
+        params = {
+            "ips": ips
+            if not isinstance(ips[0], IPAddress)
+            else [ip.address for ip in ips],
+            "linode_id": linode_id,
+        }
+
+        self._client.post("/networking/ips/share", model=self, data=params)
+
+    def ip_addresses_assign(self, assignments, region):
+        """
+        Assign multiple IPv4 addresses and/or IPv6 ranges to multiple Linodes in one Region.
+        """
+
+        for a in assignments["assignments"]:
+            if not "address" in a or not "linode_id" in a:
+                raise ValueError("Invalid assignment: {}".format(a))
+
+        if isinstance(region, Region):
+            region = region.id
+
+        params = {"assignments": assignments, "region": region}
+
+        self._client.post("/networking/ips/assign", model=self, data=params)
```

Comment:
We might want to move these functions into the networking group in `linode_client.py` since they're "global" endpoints that don't rely on a parent resource.